docs/kratos/self-service/flows/account-recovery-password-reset.mdx

```diff
@@ -1,77 +1,108 @@
 ---
 id: account-recovery-password-reset
-title: Account recovery and password reset
+title: Recover access to accounts and perform password reset
+sidebar_label: Account recovery
+toc_max_heading_level: 4
 ---
 
-```mdx-code-block
 import RenderFlow from "@site/src/theme/Code/RenderFlow"
-
 import Tabs from "@theme/Tabs"
 import TabItem from "@theme/TabItem"
-```
 
 Account recovery allows registered users to regain access to their account if they forget their password or lose access to the
 second authentication factor required for multifactor authentication (MFA).
 
 To regain access to their account, the user must prove that they are the owner of the account. The verification of ownership is
-performed using the recovery address defined by the user. This could be an email address or a phone number. When the account
-recovery flow is initiated, the system sends a link or a one-time code to the recovery address defined by the user. The user must
-access the link or enter the code they received to confirm their identity and ownership of the account.
+performed using the [recovery address defined by the user](#account-recovery-address). When the account recovery flow is
+initiated, the system sends a link or a one-time code to the recovery address defined by the user. The user must access the link
+or enter the code they received to confirm their identity and ownership of the account.
+
+When the user accesses the configured recovery method, they receive a
+[privileged session](../../session-management/overview#privileged-sessions) and are taken to their account's settings page where
+they must create a new password to regain access to their account.
+
+:::info
+
+The account recovery flow doesn't reset user's second authentication factor.
+
+:::
+
+## Supported recovery strategies
+
+Ory Identities (Ory Kratos) supports two recovery strategies:
+
+- Recovery by one-time passwords (`code`)
+- Recovery by magic links (`link`)
+
+### Comparison
+
+The "one-time password" strategy (`code`) is the default, preferred, and recommended by Ory.
```

Comment:
"Strategy" is a term internally used in Kratos. I think that term is quite abstract and might be hard to grasp for people wanting to implement Kratos in their application.

A better term is "Method" as used in the configuration as well, so we absolutely need to align the formulation here. For now, it's probably best to leave it like it is, but we should do a large scale refactoring to get rid of the term "Strategy" in favor of "Method". This is just my opinion, but I think Aeneas mentioned something like that before as well.

Reply:
We can start with this one and switch to method, not a problem for me :)